src/components/MetricsPane/charts/AggregateHabitatComplexity.jsx

```diff
@@ -0,0 +1,58 @@
+import { useContext } from 'react'
+import Plot from 'react-plotly.js'
+
+import { ChartSubtitle, ChartWrapper, TitlesWrapper } from './Charts.styles'
+import { FilterProjectsContext } from '../../../context/FilterProjectsContext'
+import { MetricCardH3 } from '../MetricsPane.styles'
+import dashboardOnlyTheme from '../../../styles/dashboardOnlyTheme'
+
+const chartTheme = dashboardOnlyTheme.plotlyChart
+
+export const AggregateHabitatComplexity = () => {
+  const { displayedSurveys } = useContext(FilterProjectsContext)
+
+  const habitatComplexityValues = displayedSurveys
+    .map((record) => record.protocols.habitatcomplexity?.score_avg_avg)
+    .filter(Boolean)
+
+  return (
+    <ChartWrapper>
+      <TitlesWrapper>
+        <MetricCardH3>Habitat Complexity </MetricCardH3>
+        <ChartSubtitle>{habitatComplexityValues.length} Surveys</ChartSubtitle>
+      </TitlesWrapper>
+
+      <Plot
+        data={[
+          {
+            x: habitatComplexityValues,
+            type: 'histogram',
+            marker: {
+              color: chartTheme.aggregateCharts.default.marker.color,
+            },
+            xbins: {
```

Comment:
same feedback as previous pr, could preprocess chart data and layout (add as `const`) for better readability